xbmc/Application.cpp

```diff
@@ -3355,21 +3354,23 @@ void CApplication::OnPlayerCloseFile(const CFileItem &file, const CBookmark &boo
 
   percent = bookmark.timeInSeconds / bookmark.totalTimeInSeconds * 100;
 
-  if ((fileItem.IsAudio() && g_advancedSettings.m_audioPlayCountMinimumPercent > 0 &&
-       percent >= g_advancedSettings.m_audioPlayCountMinimumPercent) ||
-      (fileItem.IsVideo() && g_advancedSettings.m_videoPlayCountMinimumPercent > 0 &&
-       percent >= g_advancedSettings.m_videoPlayCountMinimumPercent))
+  CAdvancedSettings& advancedSettings = CServiceBroker::GetAdvancedSettings();
```

Comment:
can this be `const CAdvancedSettings& advancedSettings`? I try to take advantage of these efforts to sprinkle some const-correctness around

Reply:
sure, will change all occurences, thanks